src/components/form-control/form-control.css

```diff
@@ -14,15 +14,9 @@
 }
 
 .required {
-  composes: label;
   margin-left: var(--spacing-small);
```

Comment:
If the label container was `inline-flex` or whatever and we used a `gap` property on it... we wouldn't even need a required classname. 💥

Reply:
Yep, you're 100% right! 

c48e1ec5